### PR TITLE
feat(script)!: be consistent with installation token naming

### PR DESCRIPTION
**Pull request by sumo-drosiek**
This PR consist of the following changes:

- deprecate `--skip-install-token` and add `--skip-installation-token`
- deprecate `SUMOLOGIC_INSTALL_TOKEN` env and add `SUMOLOGIC_INSTALLALTION_TOKEN`

### CODE REVIEW

**Comment by swiatekm-sumo** [comment]
Would be a lot of work to accept both forms for a time? It'd make the migration easier on the UI side.

**Comment by sumo-drosiek** [comment]
> Would be a lot of work to accept both forms for a time? It'd make the migration easier on the UI side.

It should accept both ways now (except windows which I'm trying to setup env for)

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 This release introduces the following breaking changes:
 
 - feat(sumologicextension): use hostname as default collector name [#918]
+- feat(script)!: be consistent with installation token naming [#941]
+
+  We renamed `--skip-install-token` to `--skip-installation-token`
+  and `SUMOLOGIC_INSTALL_TOKEN` environmental variable to `SUMOLOGIC_INSTALLATION_TOKEN`.
+  Please update your configuration and automation scripts.
```
Comment (sumo-drosiek) [inline CHANGELOG.md:17]:
```suggestion
- feat(script)!: be consistent with installation token naming [#941]

  We depracated `--skip-install-token` in favor of `--skip-installation-token`
  and `SUMOLOGIC_INSTALL_TOKEN` environmental variable in favor of `SUMOLOGIC_INSTALLATION_TOKEN`.
  Please update your configuration and automation scripts.
```

File: examples/ansible/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ This playbook will install Sumo Logic Distro of [OpenTelemetry Collector][otc_li
 
 ## Running playbook
 
-- Get an [install token][install_token] from Sumo Logic, see
+- Get an [install installation token][installation_token] from Sumo Logic, see
```
Comment (sumo-drosiek) [inline examples/ansible/README.md:7]:
```suggestion
- Get an [installation token][installation_token] from Sumo Logic, see
```

**Comment by amdprophet** [comment]
There are a few lines that need to be updated for Windows:
* https://github.com/SumoLogic/sumologic-otel-collector/blob/drosiek-installation-token/packaging/msi/wix/package.wxs#L56-L64
* https://github.com/SumoLogic/sumologic-otel-collector/blob/drosiek-installation-token/packaging/msi/wix/package.wxs#L84
* https://github.com/SumoLogic/sumologic-otel-collector/blob/drosiek-installation-token/packaging/msi/wix/ui/appdata-dir-dlg.wxs#L20
* https://github.com/SumoLogic/sumologic-otel-collector/blob/drosiek-installation-token/packaging/msi/wix/components.wxs#L25-L28

Supporting both INSTALLTOKEN and INSTALLATIONTOKEN on Windows is doable but will require adding a second WiX property, passing it to the CustomAction, and updating the DLL to support both. Are the Windows packages being used yet or can it be skipped?

**Review by swiatekm-sumo — APPROVED** [review]
The changes look fine to me. Do we also want to change the name of the config value in the Sumo extension?

**Comment by sumo-drosiek** [comment]
@swiatekm-sumo I created Issue for that: https://github.com/SumoLogic/sumologic-otel-collector/issues/965

**Review by amdprophet — APPROVED** [review]

**Pull request merged**